.gitignore

```diff
@@ -1,2 +1,11 @@
 /target/
 /src/main/resources/local.properties
+vagrant/ansible/files/qucosa-fcrepo-reportingdb-1.0.0-SNAPSHOT.war
```

Comment:
The war-file ignore needs a wildcard. Otherwise versions different from 1.0.0-SNAPSHOT are going to be reported as change.